backends/bmv2/common/action.h

```diff
@@ -25,7 +25,12 @@ namespace P4::BMV2 {
 class ActionConverter : public Inspector {
     ConversionContext *ctxt;
 
-    void convertActionBody(const IR::Vector<IR::StatOrDecl> *body, Util::JsonArray *result);
+    void convertActionBodyTop(const IR::Vector<IR::StatOrDecl> *body, Util::JsonArray *result);
+    void convertActionBody(const IR::Vector<IR::StatOrDecl> *body, Util::JsonArray *result,
```

Comment:
Nit: I'd consider making this a struct that has all this information. Makes it easier to change and read.

Reply:
Good idea.  Done in commit 7.